crates/nu-cli/src/repl.rs

```diff
@@ -472,9 +474,7 @@ fn loop_iteration(
                 Some(HistoryFileFormat::Sqlite)
             );
 
-            if history_supports_meta {
```

Comment:
I assume you didn't mean to get rid of this if statement :).

Reply:
doh! good catch. fixed.